keyboards/checkerboards/quark_lp/readme.md

```diff
@@ -0,0 +1,15 @@
+# Quark_LP
+
+![Quark_LP](https://i.imgur.com/EgdbzxE.png)
+
+The Quark LP PCB is compatible with Cherry Low-Profile switches and Kailh Choc switches, and is drop-in compatible with the OLKB Planck board. 
+
+* Keyboard Maintainer: [Nasp](https://github.com/npspears)
+* Hardware Supported: Planck
+* Hardware Availability: www.checkerboards.xyz
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
no problem - it's been added